### DISCUSSION TITLE
Link configuration

### DISCUSSION THREAD

**Discussion by Cipherdale** [YOURLS/YOURLS#2911] [General]
Dear Yourls Creator,

it is possible to create privacy link with yourls api? I mean there is no click tracking or country tracking, etc? If is possible, is there any configuration for this throuh the api?

**Comment by ozh** [comment]
There's a site wide setting to prevent logging, see http://yourls.org/#Config;optional

If you need something link based, or only with the API (not sure what you mean by this), sure it's possible : see https://github.com/YOURLS/YOURLS/wiki/Possible-With-A-Plugin

**Reply by Cipherdale** [reply]
Thanks for your answer.

i mean the API that can i make visitor to my website choose. Example: they can choose, so they can turn off on logging from the front end.
